src/content/self-hosted/administration/configuration.mdx

```diff
@@ -324,13 +324,14 @@ Use `tolerations.devPool` to limit the autoscaler analysis to a subset of cluste
 - `annotations`: Annotations to add to the autoscaler pods.
 - `cpu.up`: Increase the cluster size when the CPU consumption is greater than or equal to this value. It defaults to 60 percent.
 - `cpu.down`: Decrease the cluster size when the CPU consumption is lesser than this value. It defaults to 40 percent.
-- `increment`: The number of new nodes to request when all the current nodes are overloaded. e.g. if this value is 3, the autoscaler will request 3 new nodes when all the cluster nodes are overloaded. It defaults to 1.
+- `image`: image used by the autoscaler to deploy the autoscaler agent. It defaults to `busybox`.
 - `labels`: Labels to add to the autoscaler pods.
-- `max`: Maximum number of nodes in the cluster. It defaults to 10. Zero means unlimited.
 - `memory.up`: Increase the cluster size when the Memory consumption is greater than or equal to this value. It defaults to 70 percent.
 - `memory.down`: Decrease the cluster size when the Memory consumption is lesser than this value. It defaults to 50 percent.
-- `min`: Minimum number of nodes in the cluster. It defaults to 1.
-- `nodePoolLabel`: The node label that identifies the node pool of the node. For example, the value in GKE is `cloud.google.com/gke-nodepool`. In EKS the value is `eks.amazonaws.com/nodegroup`. If set, the autoscaler scales each node pool independently.
+- `nodes.increment`: The number of new nodes to request when all the current nodes are overloaded. e.g. if this value is 3, the autoscaler will request 3 new nodes when all the cluster nodes are overloaded. It defaults to 1.
+- `nodes.min`: Minimum number of nodes in the cluster. It defaults to 1.
+- `nodes.max`: Maximum number of nodes in the cluster. It defaults to 10. Zero means unlimited.
+- `nodes.poolLabel`: The node label that identifies the node pool of the node. For example, the value in GKE is `cloud.google.com/gke-nodepool`. In EKS the value is `eks.amazonaws.com/nodegroup`. If set, the autoscaler scales each node pool independently.
```

Comment:
Should we announce this as breaking change in the release notes?

Reply:
It's not a breaking change, the previous values also work